sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFileFormat.scala

```diff
@@ -578,62 +583,6 @@ private[sql] object ParquetFileFormat extends Logging {
     }
   }
 
-  /** This closure sets various Parquet configurations at both driver side and executor side. */
-  private[parquet] def initializeLocalJobFunc(
```

Comment:
Nor `overrideMinSplitSize()` as well. I guess https://issues.apache.org/jira/browse/SPARK-10143 issue would be still happening. I wanted to remove this after verifying it but I could have some time to do so..